### PR TITLE
Bk matplotlib

### PR DESCRIPTION
**Pull request by evanyeohboonkhai**

### CODE REVIEW

**Comment by evanyeohboonkhai** [comment]
Please have a view @KianYang-Lee  @scot-certifai

File: Matplotlib/Solution/03 - Creating different types of graphs with Pyplot.ipynb [hunk 1]
```diff
@@ -0,0 +1,849 @@
...
+    "under the License. <br>\n",
+    "<br>**SPDX-License-Identifier: Apache-2.0**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "inner-chicago",
+   "metadata": {},
+   "source": [
+    "# 02 - Creating different types of graphs with Pyplot\n",
```
Comment (KianYang-Lee) [inline Matplotlib/Solution/03 - Creating different types of graphs with Pyplot.ipynb]: Heading index different with notebook title, please correct

File: Matplotlib/Solution/04 - Creating different types of graphs with Pyplot (Exercise).ipynb [hunk 2]
```diff
@@ -0,0 +1,444 @@
...
+    "under the License. <br>\n",
+    "<br>**SPDX-License-Identifier: Apache-2.0**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "israeli-plenty",
+   "metadata": {},
+   "source": [
+    "## <a name=\"Exercise \">Exercise</a>\n",
```
Comment (KianYang-Lee) [inline Matplotlib/Solution/04 - Creating different types of graphs with Pyplot (Exercise).ipynb]: As per what discussed, please include notebook name as the heading for tilte

File: Matplotlib/Solution/02 - Getting started with Pyplot (Exercise).ipynb [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+    "under the License. <br>\n",
+    "<br>**SPDX-License-Identifier: Apache-2.0**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "offensive-model",
+   "metadata": {},
+   "source": [
+    "## <a name=\"Exercise\">Exercise</a>\n",
```
Comment (KianYang-Lee) [inline Matplotlib/Solution/02 - Getting started with Pyplot (Exercise).ipynb]: As per what discussed, please include notebook name as the heading for tilte

**Review by KianYang-Lee — COMMENTED** [review]
Code-wise is executable and content has managed to cover basics of Matplotlib. Also included some suggestions for formatting to @evanyeohboonkhai  Can you kindly take a look too @scot-certifai ? would appreciate that.

**Comment by evanyeohboonkhai** [comment]
Thanks, @KianYang-Lee  for notice the title, I made a modification to that. Please have a view.

**Comment by KianYang-Lee** [comment]
Took a view, good to go.

**Comment by evanyeohboonkhai** [comment]
@scot-certifai  if good from your side please merged it. Thanks

**Review by scotgopal — CHANGES_REQUESTED** [review]
Hi @evanyeohboonkhai . I've reviewed the code and they all seem to work fine and I didn't catch any issue with the formats. As for the grammar, I'd like to suggest you to install the Grammarly extension for Microsoft word. You may also follow the guide [here](https://towardsdatascience.com/7-essential-tips-for-writing-with-jupyter-notebook-60972a1a8901) to convert jupyter notebook to .docx. Based on the suggested corrections provided by the Grammarly extension in word, you may edit your notebook accordingly. This is the only way that we can proof read our notebooks until Grammarly supports Jupyter Notebooks.

**Comment by evanyeohboonkhai** [comment]
@scot-certifai  All the markdown is send to Grammarly and modification is done. Please have a view at it. Thanks.

File: Matplotlib/solution/01 - Getting started with Pyplot.ipynb [hunk 4]
```diff
@@ -0,0 +1,974 @@
+{
```
Comment (scotgopal) [inline Matplotlib/solution/01 - Getting started with Pyplot.ipynb:1]: I believe this 'solution' folder is mistakenly created. Can you please remove this folder @evanyeohboonkhai ? I will make the merge soon after. The rest looks good to me.

**Review by scotgopal — CHANGES_REQUESTED** [review]

**Pull request merged**